platform/linuxbsd/display_server_x11.cpp

```diff
@@ -1309,6 +1309,11 @@ int64_t DisplayServerX11::window_get_native_handle(HandleType p_handle_type, Win
 		case WINDOW_VIEW: {
 			return 0; // Not supported.
 		}
+#ifdef GLES3_ENABLED
+		case OPENGL_CONTEXT: {
+			return (int64_t)gl_manager->get_glx_context(p_window);
+		}
+#endif
```

Comment:
For consistency, it's probably should be added to the macOS `DisplayServer` to return `NSOpenGLContext *` (even if it's not relevant for OpenXR).

Reply:
That makes sense! I don't have a good way to test it, but I've added some code that should do this